content/02.introduction.md

```diff
@@ -65,25 +65,20 @@ Genomic analyses and comparisons to other known coronaviruses suggest that SARS-
 This suggests that SARS-CoV-2 may have originated in viral reservoirs of similar hosts; however, current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans [@doi:10.1038/s41591-020-0820-9].
 
-Most coronaviruses are considered zoonotic viruses with little to no transmission observed in humans.
-A major group of coronaviruses include human coronaviruses (HCoVs) strains associated with multiple respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612].
-Approximately one-third of common cold infections in humans is attributable to four out of six previously known human coronaviruses (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) that are globally circulating in the human population [@doi:10.1038/s41579-018-0118-9;@doi:10.3390/diseases4030026].
-In the past two decades, however, highly pathogenic human coronaviruses have been identified, including the severe acute respiratory syndrome coronavirus 1 (SARS-CoV-1) and the Middle East respiratory syndrome coronavirus (MERS-CoV) although both infections were confined to specific geographic regions [@doi:10.1038/nrmicro.2016.81;@doi:10.1038/s41579-018-0118-9].
-
-The current pandemic of COVID-19, caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), represents an acute and rapidly spreading global health crisis with symptoms for reported cases ranging from mild to severe or fatal [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html] and including outcomes such as acute respiratory distress, acute lung injury and other pulmonary complications.
-The possibility of asymptomatic cases is also being investigated.
 In general, respiratory viruses like coronavirus can have multiple routes of person-to-person transmission including droplet transmission (i.e. inhalation for cough, sneeze), aerosol transmission (i.e. virus suspended in air), and contact transmission (i.e. contact with oral, nasal, and eye mucous membranes).
 Other modes of transmission, such as through touching surfaces or objects and then touching mucous membranes, can also be investigated.
-While droplet-based and contact were initially considered to be the primary modes of SARS-CoV-2 transmission [@doi:10.1038/s41368-020-0075-9], as additional information has emerged, aerosol transmission has become a growing concern [@doi:10.1126/science.abc6197].
+While droplet-based and contact were initially considered to be the primary modes of SARS-CoV-2 transmission [@doi:10.1038/s41368-020-0075-9], as additional information has emerged, the possibility of aerosol transmission has also been raised [@doi:10.1126/science.abc6197].
```

Comment:
```suggestion
While droplet-based and contact were initially considered to be the primary modes of SARS-CoV-2 transmission [@doi:10.1038/s41368-020-0075-9], as additional information has emerged, the possibility of aerosol transmission has also been raised [@doi:10.1126/science.abc6197; @doi:10.1056/NEJMc2004973; @doi:10.1093/cid/ciaa939].
```

Reply:
Made the change and opened an issue for one of the papers that you referenced that didn't yet have an issue.